index.js

```diff
@@ -3,53 +3,168 @@ const state = {
     {
       id: "001-beetroot",
       name: "beetroot",
-      price: 0.35
+      price: 0.35,
+      quantity: 0,
     },
     {
       id: "002-carrot",
       name: "carrot",
-      price: 0.35
+      price: 0.35,
+      quantity: 0,
     },
     {
       id: "003-apple",
       name: "apple",
-      price: 0.35
+      price: 0.35,
+      quantity: 0,
     },
     {
       id: "004-apricot",
       name: "apricot",
-      price: 0.35
+      price: 0.35,
+      quantity: 0,
     },
     {
       id: "005-avocado",
       name: "avocado",
-      price: 0.35
+      price: 0.35,
+      quantity: 0,
     },
     {
       id: "006-bananas",
       name: "bananas",
-      price: 0.35
+      price: 0.35,
+      quantity: 0,
     },
     {
       id: "007-bell-pepper",
       name: "bell pepper",
-      price: 0.35
+      price: 0.35,
+      quantity: 0,
     },
     {
       id: "008-berry",
       name: "berry",
-      price: 0.35
+      price: 0.35,
+      quantity: 0,
     },
     {
       id: "009-blueberry",
       name: "blueberry",
-      price: 0.35
+      price: 0.35,
+      quantity: 0,
     },
     {
       id: "010-eggplant",
       name: "eggplant",
-      price: 0.35
-    }
+      price: 0.35,
+      quantity: 0,
```

Comment:
You could maybe look at adding the quantity dynamically rather than hard coding it into the state? That way you have logic already for adding more properties into the state later (type for filters, any offers being applied... etc...)